Files/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -201,11 +201,12 @@
                 <FontIcon
                     x:Name="CloudDriveSyncStatusGlyph"
                     Grid.Column="0"
+                    Width="20"
                     Margin="10,0,0,0"
-                    x:Load="{x:Bind ((local3:CloudDriveSyncStatusUI)SyncStatusUI).LoadSyncStatus, Mode=OneWay}"
```

Comment:
Does this have a performance tradeoff?

Reply:
Not that I could tell, if there is any it's minimal.
I tried to use ``x:Load``, but the compiler threw an ``Invalid parameter`` exception with the binding.

Reply:
Btw, doesn't this lead to the reverse issue where there is extra space when there is no status?

Reply:
Yes, but only in cloud drive directories (which I don't think is a bad thing because it also keeps items lined up nicely for cloud folders with shortcuts).